kivymd/uix/picker.py

```diff
@@ -1994,129 +1997,848 @@ def change_month(self, operation):
 
 Builder.load_string(
     """
-#:import CircularTimePicker kivymd.vendor.circularTimePicker.CircularTimePicker
-#:import dp kivy.metrics.dp
+<TimeInputLabel@MDLabel>:
+    theme_text_color:"Custom"
+    font_size: dp(10)
+    halign: "left"
+    valign: "bottom"
+    adaptive_size: True
 
 
-<MDTimePicker>
-    size_hint: (None, None)
-    size: (dp(270), dp(335) + dp(95))
-    pos_hint: {"center_x": .5, "center_y": .5}
+<AM_PM_Selector_Label>
+    halign: "center"
+    valign: "center"
+    theme_text_color: "Custom"
 
-    canvas:
+
+<AM_PM_Selector>
+    size_hint: None, None
+
+    canvas.before:
+        Color:
+            rgba: root.border_color
+        RoundedRectangle:
+            pos: self.pos
+            size: self.size
+            radius: [root.border_radius,]
+
+        #AM
         Color:
-            rgba: self.theme_cls.bg_light
+            rgba: root._am_bg_color
         RoundedRectangle:
-            size: (dp(270), dp(335))
-            pos: (root.pos[0], root.pos[1] + root.height - dp(335) - dp(95))
-            radius: [dp(0), dp(0), root.radius[2], root.radius[3]]
+            pos: [\
+                self.pos[0]+root.border_width,\
+                self.pos[1]+self.height/2+self.border_width*0.5\
+                ] if self.orientation=="vertical" else\
+                [\
+                self.pos[0]+root.border_width,\
+                self.pos[1]+root.border_width\
+                ]
+            size: [\
```

Comment:
Could you amend the code in the whole KV string with such an overview so that it meets the requirements of PEP8?

Wrong:

```python
            size: [\
                self.size[0]-root.border_width*2,\
                self.size[1]/2-self.border_width*1.5\
                ] if self.orientation=="vertical" else\
                [\
                self.size[0]/2-root.border_width*1.5,\
                self.size[1]-root.border_width*2\
                ]
```

Right:

```python
            size: [ \
                self.size[0] - root.border_width * 2, \
                self.size[1] / 2 - self.border_width * 1.5 \
                ] if self.orientation == "vertical" else \
                [ \
                self.size[0] / 2-root.border_width * 1.5, \
                self.size[1] - root.border_width * 2 \
                ]
```

Reply:
@HeaTTheatR Updated. sorry for mistake :)